File: freecodecamp/daily-code-challenge/2026-02-26.js
function countLettersAndNumbers(str) {
  let letters = 0;
  let numbers = 0;

  for (let char of str) {
    if (/[a-zA-Z]/.test(char)) {
      letters++;
    } else if (/[0-9]/.test(char)) {
      numbers++;
    }
  }

  const letterWord = letters === 1 ? "letter" : "letters";
  const numberWord = numbers === 1 ? "number" : "numbers";

  return `The string has ${letters} ${letterWord} and ${numbers} ${numberWord}.`;
}

File: freecodecamp/daily-code-challenge/2026-02-26.py
import re

def count_letters_and_numbers(s):
    letters = len(re.findall(r"[A-Za-z]", s))
    numbers = len(re.findall(r"[0-9]", s))

    letter_word = "letter" if letters == 1 else "letters"
    number_word = "number" if numbers == 1 else "numbers"

    return f"The string has {letters} {letter_word} and {numbers} {number_word}."
